.changeset/long-dots-cry.md

```diff
@@ -1,6 +1,6 @@
 ---
 "ui-library": minor
-"budget-it": minor
+"@inkbeard/budget-it": minor
```

Comment:
This fixes an issue in `main` where `budget-it` was updated to `@inkbeard/budget-it`.